recipes/systemc-components/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from conans import ConanFile, CMake
+from conan.tools.build import cross_building
+
+class SystemcComponentsTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def requirements(self):
+        self.requires("systemc/2.3.3")
```

Comment:
test package should only depends on your package. If it requires another requirement, something is wrong. Or you skipped important requirements from systemc-components, OR, your test package is not a test package, but an unit test or an example. Reduce it a very small usage, like, checking the library version, the idea is testing your package, not running an feature example.